### PR TITLE
Add 'VITE_' prefix to SvelteKit framework

### PR DESCRIPTION
**Pull request by domeccleston**
### 📋 Checklist

Since SvelteKit uses Vite under the hood, it needs the _VITE environment variable prefix. This adds it.

Example which reproduces the Svelte Kit example with undefined environment variables:
-  [Repo](https://github.com/domeccleston/env-var-vercel-sveltekit)
- [Preview](https://env-var-vercel-sveltekit-domeccleston.vercel.app/)

#### Tests

Currently I get an error when trying to run the tests: `  Module ts-jest in the transform option was not found.`

- [ ] The code changed/added as part of this PR has been covered with tests
- [ ] All tests pass locally with `yarn test-unit`

#### Code Review

- [x] This PR has a concise title and thorough description useful to a reviewer
- [ ] Issue from task tracker has a link to this PR

### CODE REVIEW

**Comment by leerob** [comment]
You might want to check with @styfle, I believe changing the env var preset is considered a breaking change here. Did SvelteKit change this under the hood?

File: packages/frameworks/src/frameworks.ts [hunk 1]
```diff
@@ -881,6 +881,7 @@ export const frameworks = [
       'SvelteKit is a framework for building web applications of all sizes.',
     description: 'A SvelteKit app optimized to work for serverless.',
     website: 'https://kit.svelte.dev',
+    envPrefix: 'VITE_',
```
Comment (styfle) [inline packages/frameworks/src/frameworks.ts:1108]: is this documented by sveltkit?

Also, are there plans to use SVELTE_ prefix instead? 

Is the current VITE_ prefix documented?

**Comment by raymclee** [comment]
Is it the reason I cannot get SVELTEKIT_VERCEL_URL on vercel?

File: packages/frameworks/src/frameworks.ts (hunk 1 above)
Comment (styfle) [reply packages/frameworks/src/frameworks.ts:1108]: Looks like its documented here https://kit.svelte.dev/faq#env-vars

**Review by styfle — APPROVED** [review]
Thanks!

**Review by Rich-Harris — APPROVED** [review]

**Review by TooTallNate — APPROVED** [review]

**Comment by TooTallNate** [comment]
Thanks @domeccleston!

**Comment by devunt** [comment]
This should be updated to `PUBLIC_` prefix, according to the newly released SvelteKit v1.0 docs [1]

[1] https://kit.svelte.dev/docs/configuration#env

**Comment by Rich-Harris** [comment]
We actually don't want a prefix here, because it would cause the values to be available via `$env/static/public` and `$env/dynamic/public`, when they only need to be available on the server (via `$env/static/private` and `$env/dynamic/private`).

In particular, adding them to public env will mean they're included in every rendered HTML page (because that's how `$env/dynamic/public` is populated), which is undesirable.

**Pull request merged**